saleor/graphql/webhook/mutations/webhook_update.py

```diff
@@ -52,6 +59,15 @@ class WebhookUpdateInput(graphene.InputObjectType):
         + PREVIEW_FEATURE,
         required=False,
     )
+    custom_headers = JSONString(
+        description=f"Custom headers, which will be added to HTTP request. "
+        f"There is a limitation of {HEADERS_NUMBER_LIMIT} headers per webhook "
+        f"and {HEADERS_LENGTH_LIMIT} characters per header."
+        f'Only "X-*" and "Authorization*" keys are allowed.'
+        + ADDED_IN_312
+        + PREVIEW_FEATURE,
```

Comment:
Why not use f-string?

```suggestion
        description=(
            f"Custom headers, which will be added to HTTP request. "
            f"There is a limitation of {HEADERS_NUMBER_LIMIT} headers per webhook "
            f"and {HEADERS_LENGTH_LIMIT} characters per header."
            f'Only "X-*" and "Authorization*" keys are allowed.'
            f"{ADDED_IN_312}{PREVIEW_FEATURE}"
        ),
```